doc/modules/bayes_expiry.md

```diff
@@ -5,23 +5,21 @@ title: Bayes expiry module
 
 # Bayes expiry module
 
-`Bayes expiry` module provides intelligent expiration of statistical tokens for the `new schema` of Redis statistics storage.
+The `Bayes expiry` module provides intelligent expiration of statistical tokens for the `new schema` of Redis statistics storage.
 
 ## Module configuration
 
-Configuration settings for `bayes expiry` module should be added to the corresponding `classifier` section (for instance in the `local.d/classifier-bayes.conf`).
-
-`Bayes expiry` module requires new statistics schema. It should be enabled in the classifier configuration:
+For optimal performance, the configuration settings for the `bayes expiry` module should be incorporated into the appropriate `classifier` section, such as the `local.d/classifier-bayes.conf `file. Additionally, as the `Bayes expiry` module necessitates the use of the new statistics schema, it is imperative to enable it within the classifier configuration:
```

Comment:
Optimal performance of the Bayes engine?

Reply:
The essence of the sentence is to point where the configuration directives should be written as there is no dedicated config file for the module. Sure the wrong choice of the configuration section affects the engine performance since the module won't work. I'd rather mention Bayes engine performance in the introduction. Like why do we need the `bayes expiry` module, but I'd put memory consumption on the first place.

Reply:
Yes, I do agree, maybe that should be a separate PR, as this one addresses just a style of the document and not it's semantics.